### PR TITLE
Tart up the job specs.

### PR DESCRIPTION
**Pull request by jonnyarnold**
Using [textio](https://textio.com/talent/)

### CODE REVIEW

File: jobs/developer.md [hunk 1]
```diff
@@ -4,26 +4,32 @@ title: Software Engineer
...
+**Interested?** Here’s how you apply:
+
+1. [Click here](https://jobs.lever.co/trussle/dbd3e07b-0a93-4385-a691-f188fcd80fe7/apply), upload your CV and let us know what excites you about Trussle. 
+1. We’ll arrange a phone call to give you more information about Trussle and find out more about you. We’ll ask you to complete a questionnaire and programming exercise in your own time. 
+1. If that goes well, we’ll bring you into the office for a few hours to show you around, talk more about software and meet the rest of the team. 
+1. That’s it - if we like what we see and you love what we’re doing, we’ll make you an offer.
 
-*(Not interested? Tell us why not - we’d love your feedback! Send your feedback or questions to [work@trussle.com](mailto:work@trussle.com?subject=Software%20Engineer))*
+To give everyone an equal opportunity to apply, we deliberately do not ask about your race, religion, nationality, gender, sexual orientation or age.
```
Comment (jonnywildey) [inline jobs/developer.md]: Shouldn't this be a given? Do we really need to say this?

Comment (jonnyarnold) [inline jobs/developer.md]: This was a recommendation from textio, suggesting that it is something that people look for before applying.

Comment (peter-daly) [inline jobs/developer.md]: Yeah seems like something that Uber would have to do.

Comment (jonnywildey) [inline jobs/developer.md]: I'm pretty ambivalent, if you think it help go for it

**Pull request merged**